### ISSUE TITLE
v1.14.6 release plan and checklist

### ISSUE THREAD

**Issue by patricklodder** [dogecoin/dogecoin#2975]
This issue tracks the work that needs to be done to release v1.14.6. 

If you disagree with or think I’ve missed anything, please comment and we’ll get it resolved. 

### Proposed deadlines

- [x] Deadline for any pull requests that change UI strings: 2022-06-19
- [x] Deadline for any other pull requests and translations: 2022-07-13
- [x] Deadline for any tail-end maintainer work for release preparation: 2022-07-16

### MUST HAVE (we should consider delay without these having been merged)

- [x] #2988 - helps protect wallet users with upgrades 
- [x] #3030 - fixes known Bitcoin vulnerability
- [x] #2758 - fixes potential vulnerability
- [x] #2964 - makes using the Dogecoin Core wallet cheaper and advices 3rd party devs
- [x] #2973 - we need an easy-to-use gitian script to release and to get some of the could-haves in
- [x] #2976 - enables incoming work on addr messages
- [x] #2991 - removes unmaintained code

### SHOULD HAVE (we should do our utmost best to merge these)

- [x] #2409 - dependency update
- [x] #2934 - almost ready, would be a pity to not have this
- [x] #2936 - dependency update
- [x] #2996 - increases reliability of wallet backups
- [x] #2997 - reduces unnecessary memory pressure for small nodes
- [x] #3000 - adds simple constraints to an RPC call, increases reliability 
- [x] #3001 - removes unnecessary transaction post-processing, saves resources
- [x] #3008 - removes unnecessary address processing, saves resources
- [x] #3015 - cleans up network integer types
- [x] #3037 - cleans up local sockets

### COULD HAVE (nice to haves, non-critical for release)

- [x] #2910 - cleans up warnings for compiling, non-blocking
- [x] #2974 - improves ease-of-use but otherwise non-critical

### Final translation updates

- [x] Italian: #3014
- [x] Korean: #3022
- [x] Dutch: #3026
- [x] Portuguese (`pt_PT`): #3029
- [x] Bulgarian: #3036

### Maintainer tasks for release

- [x] Refresh baseline language files - on 2022-06-22
    - #3009 - baseline
    - #3010 - recently updated translations
    - #3011 - everything else 
- [x] Update manpages: #3027
- [x] Update fixed seeds: #3038
- [x] Update checkpoints: #3031
- [x] Update copyrights: #3045
- [x] Write release notes:
    - #3035 - first draft 
    - #3043
    - #3044
    - #3046 
    - #3047
- [x] Fixate v1.14.6 in source: #3048
- [x] Tag v1.14.6
- [x] Gitian builds for v1.14.6
- [x] Fast-fwd 1.14-maint
- [x] Fast-fwd master
- [x] Upload binaries and sha256sum files

### Post-release tasks

- [x] Create 1.14.7-dev branch
- [x] Open 1.14.7-dev for development: #3057
- [x] Rebase open PRs
- [x] Delete 1.14.6-dev

**Comment by patricklodder** [comment]
Fixed year in deadlines 😂 - thanks @xanimo

**Comment by mr-fool** [comment]
Is it possible to include the file checksum in the release download page and the dogecoin website. Thank

**Comment by vertiond** [comment]
Similar to bitcoin's release process, the `.assert` files for windows/linux can be included in the release bucket along with their respective `assert.sig` files including every signature.

**Comment by vertiond** [comment]
Currently, it's not made clear to the user where to find checksums.  I don't see anything in the release description, release notes or getting started guide.

**Comment by mr-fool** [comment]
Is there an easy way to use .assert file? sha 256 checksum seems to be the most user friendly way

**Comment by patricklodder** [comment]
We can add a sha256sum file too. Adding that to the above.

**Comment by patricklodder** [comment]
Added #2991 as a must-have, this came up when looking at network hardening

**Comment by patricklodder** [comment]
- Added #2996 as should have. This improves backups (prevents situations where files get overwritten by accident)
- Removed #2957 and #2942 (nice to haves) - postponed until next release
- Deprioritized #2959 because it can use some more time to work through some of the issues we've ran into. If it gets ready in time, we can still merge it, but there's no issue if it isn't ready in time, because we postponed all dependent changes.

**Comment by patricklodder** [comment]
Added #2997 as should-have

**Comment by patricklodder** [comment]
Added #3000 and #3001 as should-have

**Comment by patricklodder** [comment]
Changed the target date for updating translation files to 6/22 because we're still working on the review of #3001

**Comment by patricklodder** [comment]
Added #3008 as should-have because it saves us processing power.

**Comment by patricklodder** [comment]
Added #3015 as should-have because it's cleanup and small

**Comment by patricklodder** [comment]
Per the above, I've set the new proposed deadline for PRs incl translations to July 13th.

**Comment by patricklodder** [comment]
Correct, and those can be deleted once you're done. We keep those in case that there is a partial change, so you can look up the old translation.

**Comment by daank-c** [comment]
> Correct, and those can be deleted once you're done. We keep those in case that there is a partial change, so you can look up the old translation.

Okay, awesome. 

Also, does the language swap mechanism in the software work similarly to how a pivot table works in something like google sheets? 🙂

(Grabs information from an array and displays it accordance with the specified parameters)

**Comment by patricklodder** [comment]
Removed #1986 and #2482(all could-haves) because they're not ready, and #2773 because it's an experimental improvement and not urgent enough. 

The only other one that I am not sure of to make it in is #2959, but I'm keeping it there for now.

**Comment by patricklodder** [comment]
Added a section to track final translation updates

**Comment by patricklodder** [comment]
All must-have items are in now - I have marked the second deadline as met. Thanks everyone for your awesome work!!!

If anyone that reads this speaks Portuguese and/or Bulgarian, we could really use your help: The Portuguese translation in #3029 and the Bulgarian translation in #3036 could really use an opinion from a native speaker.

**Comment by patricklodder** [comment]
Added #3037 to should-have because it cleans up local socket code. Removed #2959 because we need more time for that.

**Comment by patricklodder** [comment]
Added "update copyrights" maintainer task as we seem to always forget that... but not this time 😊

**Comment by patricklodder** [comment]
Removed #2726 - needs some work and we think it's best to split it off into its own repo.

**Comment by patricklodder** [comment]
All work is proposed and in a good state - ✅ 
Open pulls are approved - ✅
24-hour grace period for these final approvals is in effect - ⏳

... after that, bar anything crazy, we'll be building the release. 🥳

**Comment by patricklodder** [comment]
Tagged v1.14.6 ✅

I'm starting a gitian build on my end now (using LXC so everyone else can use docker). We will need at least 2 attested builds before we release binaries.

**Comment by chromatic** [comment]
I'll attach my signatures here:

```
bf6123a91ba2829e03dc7c48865971dd615b78249f6ee41a4e796f942bd93869  dogecoin-1.14.6-osx-unsigned.dmg
e48ac5ff72462615b3cf3fea576370aa3473383062fe711cac648fca5f6e2819  dogecoin-1.14.6-osx-unsigned.tar.gz
e3a7f1072d7763133bdb9a0be7774ce97210f3085e649cc7d103f3606ccceadf  dogecoin-1.14.6-osx64.tar.gz
e0ac42b06b6feee06f853c9c9f7657713817f993e9e00915758cf408448accee  src/dogecoin-1.14.6.tar.gz
d66c31aa4d28cef0d1ca152a4692c123b1718fc82161f3d7e4459c065377519e  dogecoin-osx-1.14-res.yml
```

**Comment by patricklodder** [comment]
```
28b6e155aabd5e5f50f520048f1bc45d73fa52637676c30e23286e30c5240e26  ./dogecoin-1.14.6-aarch64-linux-gnu-debug.tar.gz
87419c29607b2612746fccebd694037e4be7600fc32198c4989f919be20952db  ./dogecoin-1.14.6-aarch64-linux-gnu.tar.gz
36297b817e9fdbcb533575d43995a00d28648d9f2f17c09a88dd8fc39035764f  ./dogecoin-1.14.6-arm-linux-gnueabihf-debug.tar.gz
d0b7f5f4fbabb6a10078ac9cde1df7eb37bef4c2627cecfbf70746387c59f914  ./dogecoin-1.14.6-arm-linux-gnueabihf.tar.gz
df5ea74f46b2910fa6ece56c0b6099258b62f57e55175db4d7b0b52047d6d03b  ./dogecoin-1.14.6-i686-pc-linux-gnu-debug.tar.gz
3e60c4c818cb44abcca5b3bf9eff6baf86834c762e41d886c19bd721c00d0e24  ./dogecoin-1.14.6-i686-pc-linux-gnu.tar.gz
ef418bc4fd6ef69f60da37432d2b52f88c3703f666a8ab0edb0f7f8a2c9d6d28  ./dogecoin-1.14.6-x86_64-linux-gnu-debug.tar.gz
b9c5480cfd4fd34e524e057d04f6990679f2c83be5a868dab3aaaa8aa85b189c  ./dogecoin-1.14.6-x86_64-linux-gnu.tar.gz
e0ac42b06b6feee06f853c9c9f7657713817f993e9e00915758cf408448accee  ./dogecoin-1.14.6.tar.gz
```

**Comment by patricklodder** [comment]
```
ef418bc4fd6ef69f60da37432d2b52f88c3703f666a8ab0edb0f7f8a2c9d6d28  ./dogecoin-1.14.6-x86_64-linux-gnu-debug.tar.gz
b9c5480cfd4fd34e524e057d04f6990679f2c83be5a868dab3aaaa8aa85b189c  ./dogecoin-1.14.6-x86_64-linux-gnu.tar.gz
```

vs

```
a3d442b3b28d22ee08f208deb0e6a922975ad9ef3f32355de54094847981f98f  dogecoin-1.14.6-x86_64-linux-gnu-debug.tar.gz
fe9c9cdab946155866a5bd5a5127d2971a9eed3e0b65fb553fe393ad1daaebb0  dogecoin-1.14.6-x86_64-linux-gnu.tar.gz
```

That's a mismatch.

I'll re-run my build.

**Comment by patricklodder** [comment]
Re-built on new box:

```
28b6e155aabd5e5f50f520048f1bc45d73fa52637676c30e23286e30c5240e26  dogecoin-1.14.6-aarch64-linux-gnu-debug.tar.gz
87419c29607b2612746fccebd694037e4be7600fc32198c4989f919be20952db  dogecoin-1.14.6-aarch64-linux-gnu.tar.gz
36297b817e9fdbcb533575d43995a00d28648d9f2f17c09a88dd8fc39035764f  dogecoin-1.14.6-arm-linux-gnueabihf-debug.tar.gz
d0b7f5f4fbabb6a10078ac9cde1df7eb37bef4c2627cecfbf70746387c59f914  dogecoin-1.14.6-arm-linux-gnueabihf.tar.gz
df5ea74f46b2910fa6ece56c0b6099258b62f57e55175db4d7b0b52047d6d03b  dogecoin-1.14.6-i686-pc-linux-gnu-debug.tar.gz
3e60c4c818cb44abcca5b3bf9eff6baf86834c762e41d886c19bd721c00d0e24  dogecoin-1.14.6-i686-pc-linux-gnu.tar.gz
bf6123a91ba2829e03dc7c48865971dd615b78249f6ee41a4e796f942bd93869  dogecoin-1.14.6-osx-unsigned.dmg
e48ac5ff72462615b3cf3fea576370aa3473383062fe711cac648fca5f6e2819  dogecoin-1.14.6-osx-unsigned.tar.gz
e3a7f1072d7763133bdb9a0be7774ce97210f3085e649cc7d103f3606ccceadf  dogecoin-1.14.6-osx64.tar.gz
79d40c68fbf3040d8aae87a24a856cbaf732dd63851e77cf66693430ff19fe98  dogecoin-1.14.6-win32-debug.zip
c3dd01612f70afaf7b293199e016feeb8c5071fbb673a02a3baf24d04014cc02  dogecoin-1.14.6-win32-setup-unsigned.exe
c919fdc966764ec554273791699426448eedd8d305e76284e4f1cd1d5f0b5a7a  dogecoin-1.14.6-win32.zip
9d3039fa33afe7886e7761dcd735745ac870e57288328e47dddb4a24ca81889c  dogecoin-1.14.6-win64-debug.zip
888429200b5cd1a414e33215e8e8aea0957fa7a615c407b19b8a9ab2fa9e180a  dogecoin-1.14.6-win64-setup-unsigned.exe
709490ac8464b015266884831a2b5b594efc8b2c17a7e6b85255058cbee049de  dogecoin-1.14.6-win64.zip
a3d442b3b28d22ee08f208deb0e6a922975ad9ef3f32355de54094847981f98f  dogecoin-1.14.6-x86_64-linux-gnu-debug.tar.gz
fe9c9cdab946155866a5bd5a5127d2971a9eed3e0b65fb553fe393ad1daaebb0  dogecoin-1.14.6-x86_64-linux-gnu.tar.gz
e0ac42b06b6feee06f853c9c9f7657713817f993e9e00915758cf408448accee  dogecoin-1.14.6.tar.gz
```

This looks good, I'm going to process sigs and publish over at [gitian.sigs](https://github/dogecoin/gitian.sigs), will supply link when done.

**Comment by xanimo** [comment]
```
bf6123a91ba2829e03dc7c48865971dd615b78249f6ee41a4e796f942bd93869  dogecoin-1.14.6-osx-unsigned.dmg
e48ac5ff72462615b3cf3fea576370aa3473383062fe711cac648fca5f6e2819  dogecoin-1.14.6-osx-unsigned.tar.gz
e3a7f1072d7763133bdb9a0be7774ce97210f3085e649cc7d103f3606ccceadf  dogecoin-1.14.6-osx64.tar.gz
e0ac42b06b6feee06f853c9c9f7657713817f993e9e00915758cf408448accee  src/dogecoin-1.14.6.tar.gz
2faf4b06a4e7b28358de0bce20748e3f4355c1ca88e13521179209a319beb2e5  dogecoin-osx-1.14-res.yml
c16054edea300152e4f52068c6dbecd84e40c88330af74eb9bdadc7b0d0f58f6  dogecoin-1.14.6-win-unsigned.tar.gz
79d40c68fbf3040d8aae87a24a856cbaf732dd63851e77cf66693430ff19fe98  dogecoin-1.14.6-win32-debug.zip
c3dd01612f70afaf7b293199e016feeb8c5071fbb673a02a3baf24d04014cc02  dogecoin-1.14.6-win32-setup-unsigned.exe
c919fdc966764ec554273791699426448eedd8d305e76284e4f1cd1d5f0b5a7a  dogecoin-1.14.6-win32.zip
9d3039fa33afe7886e7761dcd735745ac870e57288328e47dddb4a24ca81889c  dogecoin-1.14.6-win64-debug.zip
888429200b5cd1a414e33215e8e8aea0957fa7a615c407b19b8a9ab2fa9e180a  dogecoin-1.14.6-win64-setup-unsigned.exe
709490ac8464b015266884831a2b5b594efc8b2c17a7e6b85255058cbee049de  dogecoin-1.14.6-win64.zip
e0ac42b06b6feee06f853c9c9f7657713817f993e9e00915758cf408448accee  src/dogecoin-1.14.6.tar.gz
1668245e33f1c3d27fe6e7fb6889a2ba5700da79a8438c1584ccd15bd5b580b4  dogecoin-win-1.14-res.yml
28b6e155aabd5e5f50f520048f1bc45d73fa52637676c30e23286e30c5240e26  dogecoin-1.14.6-aarch64-linux-gnu-debug.tar.gz
87419c29607b2612746fccebd694037e4be7600fc32198c4989f919be20952db  dogecoin-1.14.6-aarch64-linux-gnu.tar.gz
36297b817e9fdbcb533575d43995a00d28648d9f2f17c09a88dd8fc39035764f  dogecoin-1.14.6-arm-linux-gnueabihf-debug.tar.gz
d0b7f5f4fbabb6a10078ac9cde1df7eb37bef4c2627cecfbf70746387c59f914  dogecoin-1.14.6-arm-linux-gnueabihf.tar.gz
df5ea74f46b2910fa6ece56c0b6099258b62f57e55175db4d7b0b52047d6d03b  dogecoin-1.14.6-i686-pc-linux-gnu-debug.tar.gz
3e60c4c818cb44abcca5b3bf9eff6baf86834c762e41d886c19bd721c00d0e24  dogecoin-1.14.6-i686-pc-linux-gnu.tar.gz
a3d442b3b28d22ee08f208deb0e6a922975ad9ef3f32355de54094847981f98f  dogecoin-1.14.6-x86_64-linux-gnu-debug.tar.gz
fe9c9cdab946155866a5bd5a5127d2971a9eed3e0b65fb553fe393ad1daaebb0  dogecoin-1.14.6-x86_64-linux-gnu.tar.gz
e0ac42b06b6feee06f853c9c9f7657713817f993e9e00915758cf408448accee  src/dogecoin-1.14.6.tar.gz
6f65292cd451ffa98ffc4f377f099dabdbf2bc15750307a9896d5b0c80888992  dogecoin-linux-1.14-res.yml
```

**Comment by xanimo** [comment]
:partying_face: :rocket: here's sigs:
https://github.com/dogecoin/gitian.sigs/pull/29

**Comment by michilumin** [comment]
Inprocess!

```
bf6123a91ba2829e03dc7c48865971dd615b78249f6ee41a4e796f942bd93869  dogecoin-1.14.6-osx-unsigned.dmg
e48ac5ff72462615b3cf3fea576370aa3473383062fe711cac648fca5f6e2819  dogecoin-1.14.6-osx-unsigned.tar.gz
e3a7f1072d7763133bdb9a0be7774ce97210f3085e649cc7d103f3606ccceadf  dogecoin-1.14.6-osx64.tar.gz
e0ac42b06b6feee06f853c9c9f7657713817f993e9e00915758cf408448accee  src/dogecoin-1.14.6.tar.gz
f97005946fac85d3fca771f29ed73990b1f88b37315767b0ac441ee43a7bc7e7  dogecoin-osx-1.14-res.yml

```

... to be continued

**Comment by alamshafil** [comment]
Doing this for the first time, here are my signatures:

```
bf6123a91ba2829e03dc7c48865971dd615b78249f6ee41a4e796f942bd93869  dogecoin-1.14.6-osx-unsigned.dmg
e48ac5ff72462615b3cf3fea576370aa3473383062fe711cac648fca5f6e2819  dogecoin-1.14.6-osx-unsigned.tar.gz
e3a7f1072d7763133bdb9a0be7774ce97210f3085e649cc7d103f3606ccceadf  dogecoin-1.14.6-osx64.tar.gz
e0ac42b06b6feee06f853c9c9f7657713817f993e9e00915758cf408448accee  src/dogecoin-1.14.6.tar.gz
b13185610af8268aed66a7caab597baa013657994a60758d00d203196a6b50c4  dogecoin-osx-1.14-res.yml
c16054edea300152e4f52068c6dbecd84e40c88330af74eb9bdadc7b0d0f58f6  dogecoin-1.14.6-win-unsigned.tar.gz
79d40c68fbf3040d8aae87a24a856cbaf732dd63851e77cf66693430ff19fe98  dogecoin-1.14.6-win32-debug.zip
c3dd01612f70afaf7b293199e016feeb8c5071fbb673a02a3baf24d04014cc02  dogecoin-1.14.6-win32-setup-unsigned.exe
c919fdc966764ec554273791699426448eedd8d305e76284e4f1cd1d5f0b5a7a  dogecoin-1.14.6-win32.zip
9d3039fa33afe7886e7761dcd735745ac870e57288328e47dddb4a24ca81889c  dogecoin-1.14.6-win64-debug.zip
888429200b5cd1a414e33215e8e8aea0957fa7a615c407b19b8a9ab2fa9e180a  dogecoin-1.14.6-win64-setup-unsigned.exe
709490ac8464b015266884831a2b5b594efc8b2c17a7e6b85255058cbee049de  dogecoin-1.14.6-win64.zip
e0ac42b06b6feee06f853c9c9f7657713817f993e9e00915758cf408448accee  src/dogecoin-1.14.6.tar.gz
b1d68bd09dc14169b14e545ea9f80147eaeed7bdd9ab9629d5de4704a2dd4c10  dogecoin-win-1.14-res.yml
```

**Comment by patricklodder** [comment]
@xanimo's sigs are in, mine are pending in dogecoin/gitian.sigs#30

I'll do the macOs signing tomorrow - will have to see if attestation is going to work. If not, we'll have to figure that out for 1.14.7

For windows, I hope to finally have my windows certs tomorrow too. If not I will report back here.

**Comment by patricklodder** [comment]
> For windows, I hope to finally have my windows certs tomorrow too. If not I will report back here.

The current reason why my cert hasn't arrived is because Sectigo has for some reason changed the country code for my phone number for dev validation, so they're trying to call me in the wrong country (continent even 😂) I'm dealing with (=waiting on) their support right now.

Worst case, we'll append signed windows binaries to the release files afterwards - wouldn't be a regression as there have never been signed windows binaries.

MacOS - simple signed binaries are going to be a go, am currently trying to find a way to get it notarized through Xcode, to make installation more user-friendly. Expect to have at least some results later.

**Comment by patricklodder** [comment]
Windows status: still waiting on Sectigo support
macOS status: 
- signing works
- notarization seems to work
- deterministic signing does not work (but I learned today that it never has for 1.14 - will open a separate issue)

### PROPOSAL

At this point, we're at the same deliverables as we had with 1.14.5 for windows, and the signed Mac binary is improved but still not deterministic. To make a **deterministic** macOS build with signing and notarization, we'll need to do some changes to the gitian and macdeploy scripts, and probably have to integrate [signapple](https://github.com/achow101/signapple) instead of the current process, so here's my proposal:

1. Publish `*-osx-*` binaries, both unsigned deterministic and signed+notarized non-deterministic.
2. Publish unsigned `*-win*` binaries as we've done so far
3. Work on the signing process for macOS inside 1.14.7 together with everything else
4. If my Windows cert arrives this week (getting less likely as time passes) AND there isn't a similar issue with the detached windows signing, we add signed windows binaries to the release as soon as they are available. If not, we do a patch release 1.14.6.1 where we provide signed binaries for everything.  

This can be executed right now and the release can be published as soon as we have a form of consensus on this.

Please share your thoughts.

**Comment by xanimo** [comment]
> Windows status: still waiting on Sectigo support macOS status:
> 
>     * signing works
> 
>     * notarization seems to work
> 
>     * deterministic signing does not work (but I learned today that it never has for 1.14 - will open a separate issue)
> 
> 
> ### PROPOSAL
> 
> At this point, we're at the same deliverables as we had with 1.14.5 for windows, and the signed Mac binary is improved but still not deterministic. To make a **deterministic** macOS build with signing and notarization, we'll need to do some changes to the gitian and macdeploy scripts, and probably have to integrate [signapple](https://github.com/achow101/signapple) instead of the current process, so here's my proposal:
> 
>     1. Publish `*-osx-*` binaries, both unsigned deterministic and signed+notarized non-deterministic.
> 
>     2. Publish unsigned `*-win*` binaries as we've done so far
> 
>     3. Work on the signing process for macOS inside 1.14.7 together with everything else
> 
>     4. If my Windows cert arrives this week (getting less likely as time passes) AND there isn't a similar issue with the detached windows signing, we add signed windows binaries to the release as soon as they are available. If not, we do a patch release 1.14.6.1 where we provide signed binaries for everything.
> 
> 
> This can be executed right now and the release can be published as soon as we have a form of consensus on this.
> 
> Please share your thoughts.

This sounds sensible to me as you all have hit the baseline achieved for past releases so if anything the deterministic signed + notarized macOS binaries would be an added bonus at this point.

Re signed windows binaries, I'd opt for adding them to release as soon as they're available granted you don't encounter similar issues in the gitian build process as oppose to a 1.14.6.1 patch release.

Thanks for the update! :heart:

**Comment by michilumin** [comment]
Ok so as I mentioned to Patrick otherwise -- from experience I do not have much faith in Sectigo providing a cert in a timely manner. That's just how their ship runs; I've been through the process at least 3 times and every time it was error prone and incredibly slow. 

So while I am eager to see the windows build signed, I think that being at the mercy of waiting for Sectigo to go through their process. We can always add the signed release.

macOS, I think making both available will be good and, the unsigned deterministic will be available for those who care, and can compare the sigs on their own and know how to run it; whereas most laypersons will likely just go for the signed build and have a better experience.  So I think that's suitable.

**Comment by alamshafil** [comment]
I have no issue with the stated proposal since we have the binaries ready based on past releases. My comments below are mostly the same as stated above.

I have no problem releasing both macOS binaries, but the only minor issue I have is that the average person may be confused between a "deterministic" build and a "notarized" build. 

I personally would prefer releasing signed windows binaries; however, it looks like Sectigo is causing issues. Since the last release only contained unsigned binaries, I have no problem releasing unsigned binaries since it won't be a regression. As already stated, we can add the signed release later.

**Comment by chromatic** [comment]
:+1: to the proposal from me.

**Comment by MishaBoar** [comment]
I agree with the proposal as we have covered our past deliverables! 

Regarding the confusion @alamshafil is mentioning with macOS binaries, most people that tend to be confused will likely download the release from https://dogecoin.com/. For those using Github and having doubts, we can just provide the support through the usual channels.

Regarding Sectigo: I struggled with them for years with website certificates. It can take time, especially if there is some bureaucratic hiccup (not only, though). I could occasionally speed things up by contacting them through their web chats, which seem to work better than their phone numbers, for some reason.

Thanks all for the great work, you are awesome!
